net-dialup/accel-ppp/files/accel-ppp-1.12.0-ucontext.patch

```diff
@@ -0,0 +1,30 @@
+# Use ucontext, this is specific to musl libc
+#
+# Alpine patch:
+# https://git.alpinelinux.org/aports/tree/community/accel-ppp/0001-ucontext.patch
+#
+# Closes: https://bugs.gentoo.org/828906
+--- a/accel-pppd/CMakeLists.txt
++++ b/accel-pppd/CMakeLists.txt
+@@ -113,7 +113,21 @@ ADD_EXECUTABLE(accel-pppd
+ 	main.c
+ )
+
++option(USE_LIBUCONTEXT "Use libucontext on non glibc systems" OFF)
++IF(USE_LIBUCONTEXT)
++	find_library(LIBUCONTEXT_LIBRARY ucontext)
++
++	IF(NOT LIBUCONTEXT_LIBRARY)
++		message(FATAL_ERROR "${Esc}[31mRequired libucontext not found.\n Install ucontext and run cmake again${Esc}[m")
```

Comment:
It's a nice touch, but i wouldn't bother with the colours.

Reply:
I was thinking of upstream ing the patch hence stuck with what they were using